src/H5WebViewer.ts

```diff
@@ -8,10 +8,10 @@ import {
   window,
   workspace,
 } from 'vscode';
-import { join, basename } from 'path';
-import { writeFileSync, watchFile, unwatchFile } from 'fs';
+import { join, basename, dirname } from 'node:path';
+import { writeFileSync, watchFile, unwatchFile } from 'node:fs';
```

Comment:
This is the new recommended syntax for importing built-in Node modules: https://nodejs.org/api/esm.html#node-imports

Reply:
Much clearer, I like it.